src/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { BrowserWindow } from 'electron'
+
+export function getWindow(): BrowserWindow {
+  return BrowserWindow.getAllWindows()[0]
```

Comment:
The first window is the main window where `gmail.com` is loaded right? Should we name the function `getMainWindow()` instead?

Reply:
Yes, the first window is the one loaded when the app starts.  I will rename the function